qt/python/mantidqt/mantidqt/widgets/workspacedisplay/matrix/test/test_matrixworkspacedisplay_table_view_model.py

```diff
@@ -596,7 +596,7 @@ def test_set_unicode_unit_label(self):
         """
         ws = CreateWorkspace(DataX=[0, 1, 2], DataY=[3, 7, 5], DataE=[0.2, 0.3, 0.1], NSpec=1)
         label_unit = ws.getAxis(0).setUnit("Label")
-        microseconds = "\u00B5s"
+        microseconds = "\u00b5s"
```

Comment:
I'm not sure about this one

Reply:
I don't have any preference, but changing it back is going to be automatically changed by ruff-format